src/neofs_testlib/env/env.py

```diff
@@ -109,11 +109,14 @@ def deploy_inner_ring_node(self):
         self.inner_ring_nodes.append(new_inner_ring_node)
 
     @allure.step("Deploy storage node")
-    def deploy_storage_nodes(self, count=1):
+    def deploy_storage_nodes(self, count=1, attrs: Optional[dict] = None):
```

Comment:
it may be not very connvenient to use with raw dictionary imo, and with separated count. Some settings like LOCODE are quite interpretable

Reply:
example is needed, not sure if I understand you correctly